features/hide-footer/style.css

```diff
@@ -0,0 +1,6 @@
+#footer, #donor {
+    display: none;
+}
+.container {
+    padding-bottom: 0px !important;
+}
```

Comment:
This selects the navbar- did you mean to do that?

Reply:
Whait it does?

Reply:
Wich one? The #footer, #donor or the .container?

Reply:
Oh you meant the container. That is not the NavBar, right? I did it so stuff looks a little better

Reply:
> Oh you meant the container. That is not the NavBar, right? I did it so stuff looks a little better

It's a class name, and it also selects part of the navigation bar on 2.0 pages.

Reply:
Does that create any problems? (I can't check right now)

Reply:
Yes, it does. You generally want to be very specific when it comes to specifying stuff like this (such as IDs, classes, etc) otherwise it messes up stuff it shouldn't. 

> Does that create any problems? (I can't check right now)

Reply:
I will fix it tomorrow then

Reply:
This is fixed now :)